UefiPayloadPkg/Library/PlatformBootManagerLib/PlatformBootManager.c

```diff
@@ -1039,6 +1032,26 @@ PrintSolStrings (
   }
 }
 
+/**
+  Refresh the logo on ReadyToBoot event. It will clear the screen from strings
+
+  and progress bar when timeout is reached or continue key is pressed.
+
+  @param    Event          Event pointer.
+  @param    Context        Context pass to this function.
+**/
+VOID
+EFIAPI
+RefreshLogo (
+  IN EFI_EVENT    Event,
+  IN VOID         *Context
+  )
+{
+  gBS->CloseEvent (Event);
+  gST->ConOut->ClearScreen (gST->ConOut);

```

Comment:
This causes a short flcker before the logo is displayed again. Would be nice if we could do this without flickering

Reply:
You are asking for too much. Flickering was also present earlier if you did not press any button.

Reply:
Ok, it was more of a "Would Be Nice" thing than a request for change.